outboundSender.go

```diff
@@ -351,6 +361,15 @@ func (obs *CaduceusOutboundSender) RetiredSince() time.Time {
 	return deliverUntil
 }
 
+func contains(str string, sl []string) bool {
```

Comment:
Instead of taking a `string`, and `[]string`, you can make a function `overlaps()` that takes two `[]string`s and returns `true` when there is at least one shared string value.